client/checkout/utils/test/upe.test.js

```diff
@@ -126,6 +127,66 @@ describe( 'UPE checkout utils', () => {
 		} );
 	} );
 
+	describe( 'hasPaymentMethodCountryRestrictions', () => {
+		let container;
+
+		beforeAll( () => {
+			container = document.createElement( 'div' );
+			container.innerHTML = `
+				<ul class="wc_payment_methods payment_methods methods">
+					<li class="wc_payment_method payment_method_woocommerce_payments_card" data-payment-method-type="card">
+						<input id="payment_method_woocommerce_payments" type="radio" class="input-radio">
+					</li>
+					<li class="wc_payment_method payment_method_woocommerce_payments_bancontact" data-payment-method-type="bancontact">
+						<input id="payment_method_woocommerce_payments_bancontact" type="radio" class="input-radio">
+					</li>
+				</ul>
+			`;
+			document.body.appendChild( container );
+		} );
+
+		afterAll( () => {
+			document.body.removeChild( container );
+			container = null;
+		} );
+
+		beforeEach( () => {
+			jest.clearAllMocks();
+			getUPEConfig.mockImplementation( ( argument ) => {
+				if ( argument === 'paymentMethodsConfig' ) {
+					return {
+						card: { countries: [] },
+						bancontact: { countries: [ 'BE' ] },
+					};
+				}
+
+				if ( argument === 'gatewayId' ) {
+					return 'woocommerce_payments';
+				}
```

Comment:
Do you think we could skip configuring the `getUPEConfig` mock for this argument? 👀

Reply:
Yes, we should be able to skip it, nice catch. I removed it in 561ac89e7e08b8f43bb1a240c4979c78097c2212